lib/maven_project.rb

```diff
@@ -58,7 +58,7 @@ def compiled?
 
   def compile!
     Dir.chdir(path) do
-      SystemCommands.sh!('mvn', '-q', 'package')
+      SystemCommands.sh!('mvn -q package -Dmaven.test.skip=true', {escape: false})
```

Comment:
the escape false is needed for the `-Dmaven.test.skip=true` -- it would escape the `=` -> `\=`. I'll see if I could put it to env so that we don't need to explicitly add it for each project.